<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPlaylistWizard</class>
 <widget class="QWizard" name="ExportPlaylistWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight|QWizard::NoBackButtonOnLastPage|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="optionPage">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="mainOption">
      <property name="title">
       <string>How to generate playlists ?</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="multiPlaylist">
         <property name="text">
          <string>One playlist by folder</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="uniquePlaylist">
         <property name="text">
          <string>One playlist for all selected tracks</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QWidget" name="outputPathSelection" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputPath"/>
          </item>
          <item>
           <widget class="QToolButton" name="selectPathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="mainOptionHelp">
         <property name="text">
          <string notr="true">Help text&lt;br/&gt;on 2 lines</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="ConsoleWidget" name="outputConsole"/>
    </item>
   </layout>
  </widget>
  <action name="actionSelect_playlist_name">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/img/actions/playlist</normaloff>:/img/actions/playlist</iconset>
   </property>
   <property name="text">
    <string>Select playlist name</string>
   </property>
   <property name="toolTip">
    <string>Select playlist name</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>gui\consolewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
